src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -2335,18 +2335,17 @@ class CommonVisitor : public AST::BaseVisitor<Struct> {
         }
 
         cast_helper(left, right, false);
-        Vec<ASR::call_arg_t> args;
-        args.reserve(al, 2);
-        ASR::call_arg_t arg1, arg2;
-        arg1.loc = left->base.loc;
-        arg1.m_value = left;
-        args.push_back(al, arg1);
-        arg2.loc = right->base.loc;
-        arg2.m_value = right;
-        args.push_back(al, arg2);
-        Vec<ASR::restriction_arg_t*> rt_args;
-        rt_args.reserve(al, 1);
```

Comment:
I am not sure if this is the right thing to do. Would confirm from @certik @ansharlubis

Reply:
@Smit-create What would happen if you restore this piece of code? Will your PR stop working?

Reply:
Nope, it was an unused variable so I removed that.

Reply:
I see. If the presence of this code generates a compiler warning then keep it removed, otherwise keep it.

Reply:
> I don't see any unused variable warning with the presence of this code

That is because we have a line `rt_args.reserve(al, 1);`.

Reply:
I see. Makes sense.